<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParametersWindow</class>
 <widget class="QWidget" name="ParametersWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>171</width>
     <height>228</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="1">
       <widget class="QSpinBox" name="requestsSpinBox"/>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="bufferSpinBox"/>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="clientsSpinBox"/>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="serversSpinBox"/>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="aDoubleSpinBox"/>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="bDoubleSpinBox"/>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="lambdaDoubleSpinBox"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>requests</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>buffer</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>clients</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>servers</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>a</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>b</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>lambda</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="startButton">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
